services/github/github-actions.service.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { renderBuildStatusBadge } = require('../build-status')
+const { nonNegativeInteger } = require('../validators')
+const { GithubAuthV3Service } = require('./github-auth-service')
+const { documentation, errorMessagesFor } = require('./github-helpers')
+const { NotFound } = require('..')
+
+const schema = Joi.object({
+  total_count: nonNegativeInteger,
+  check_suites: Joi.array()
+    .items(
+      Joi.object({
+        status: Joi.string().required(),
```

Comment:
We _could_ probably try to use a stricter validation here for status and conclusion, although I've intentionally kept these at `Joi.string()`. AFAICT, GitHub apps can update the list of possible status values that they use, which would flow through here (since this API returns all Check Suites, not just GH Actions Check Suites). 

I believe that if we tried to restrict the allowed strings for these fields that we'd inevitably end up with users receiving "invalid response data" badges

Reply:
yes I think GH Actions is sufficiently flexible that its hard to reliably require something more restrictive here